Bottom: c70028eff5c90e006ed877258bd0549141c5ae50
Top:    60fcd04de3152c013703d4e15b508619dedd52a2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-21 23:45:22 +0000

Batch size for accumulating lines down to 1

This works around a bug seen on one of my boards, and is similar
to what I had in the non-pours variant of the code.


---

diff --git a/src/pour.c b/src/pour.c
index d1099cd..d9ee86f 100644
--- a/src/pour.c
+++ b/src/pour.c
@@ -68,7 +68,7 @@ RCSID ("$Id$");
 
 #define UNSUBTRACT_BLOAT 100
 #define SUBTRACT_PIN_VIA_BATCH_SIZE 100
-#define SUBTRACT_LINE_BATCH_SIZE 20
+#define SUBTRACT_LINE_BATCH_SIZE 1
 
 /* ---------------------------------------------------------------------------
  * local prototypes
